llap-server/src/java/org/apache/hadoop/hive/llap/daemon/impl/AMReporter.java

```diff
@@ -431,6 +437,15 @@ protected Void callInternal() {
 
       return null;
     }
+
+    private long getUsedMemory() {
```

Comment:
Cool  stuff, would it make more sense to present it as used percentage? We could even print both values.